test/puppetlabs/puppetdb/amq_migration_test.clj

```diff
@@ -134,8 +134,8 @@
                         (:transaction_uuid (first results))))))))))))
 
   (testing "an upgrade with a bad catalog"
-    (with-log-suppressed-unless-notable (every-pred tlog/critical-errors
-                                                    (tlog/starting-with "Output of coerce-to-new-command does not"))
+    (tlog/with-log-suppressed-unless-notable (every-pred log-utils/notable-pdb-event?
+                                                    (log-utils/starting-with "Output of coerce-to-new-command does not"))
```

Comment:
I'd adjust the indentation here.  Normally the `(log-utils` should line up with the `log-utils` above it, but this line's also awfully long.  While we're not strict about 80-columns, we generally try to keep them closer to that when possible, and (excepting internationalization-related issues) usually try to at least stay under about 100.

So I'd probably either just bind that every-pred in a let `(let [foo (every-pred ...` and use foo in the with, or even just

```clojure
    (with-log-suppressed-unless-notable
      (every-pred tlog/critical-errors
                  (tlog/starting-with "Output of coerce-to-new-command does not"))
      (with-test-db
```
Personally, I'd likely also keep the `:refer` for that symbol -- I sometimes favor unprefixed with- forms, but if you prefer the prefixed version, that's fine.